trie/utils/verkle.go

```diff
@@ -209,11 +209,7 @@ func codeChunkIndex(chunk *uint256.Int) (*uint256.Int, byte) {
 		chunkOffset            = new(uint256.Int).Add(codeOffset, chunk)
 		treeIndex, subIndexMod = new(uint256.Int).DivMod(chunkOffset, verkleNodeWidth, new(uint256.Int))
 	)
-	var subIndex byte
-	if len(subIndexMod) != 0 {
-		subIndex = byte(subIndexMod[0])
-	}
-	return treeIndex, subIndex
+	return treeIndex, byte(subIndexMod[0])
```

Comment:
I agree that the `len`-check is not needed, but the `byte(subIndexMod[0])` is IMO even worse, because it reaches in and makes assumptions about the structure of an `uint256.Int`. 
This would be better, imo: 
```suggestion
	return treeIndex, byte(subIndexMod.Uint64())
```